apps/prairielearn/src/pages/administratorBatchedMigrations/administratorBatchedMigrations.html.ts

```diff
@@ -1,33 +1,18 @@
-// @ts-check
-const { html } = require('@prairielearn/html');
-const { renderEjs } = require('@prairielearn/html-ejs');
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import {
+  BatchedMigrationJobRow,
+  BatchedMigrationRow,
+  BatchedMigrationStatus,
+} from '@prairielearn/migrations';
```

Comment:
We typically use [type-only imports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html) when we can:

```suggestion
import type {
  BatchedMigrationJobRow,
  BatchedMigrationRow,
  BatchedMigrationStatus,
} from '@prairielearn/migrations';
```

Reply:
Resolved by commit 243caf3.